iothub/device/src/IotHubBaseClient.cs

```diff
@@ -84,13 +84,15 @@ public abstract class IotHubBaseClient : IDisposable
         /// callback, you'll need to handle exceptions that could originate in there.
         /// </remarks>
         /// <example>
+        /// <code language="csharp">
```

Comment:
Requires a code tag to render the code properly, and a csharp language property to colorize it right.